server/src/sim.rs

```diff
@@ -85,6 +88,9 @@ impl Sim {
         let mut tx = save.write()?;
         let mut writer = tx.get()?;
         for (_, (pos, ch)) in self.world.query::<(&Position, &Character)>().iter() {
+            if !ch.state.active {
```

Comment:
Why shouldn't we save inactive characters?

Reply:
While it's good to save the entity itself to avoid it disappearing next time its node is saved, we don't need to save the inactive character's path from the origin every frame because the character doesn't move, so we just end up overwriting data with itself. Since paths could get pretty long, this seemed potentially worth optimizing.

It's at least worth a code comment, but I don't feel strongly here, so I am happy to revert this part (although it would result in more code if I do the `InactiveCharacter` wrapping).

Reply:
Actually, thinking about this again, this feels like a premature optimization. I could imagine later wanting to add a feature where an admin can move logged-out players around, and forgetting about this could result in unexpected desynchronization. I'll get rid of it.